packages/aws-cdk-lib/aws-rds/adr/aurora-serverless-v2.md

```diff
@@ -0,0 +1,233 @@
+
+1. How low can the instance scale down. If there is no traffic to the cluster
+   this is the minimum capacity that is allowed.
+2. How fast can the instance scale up. The scaling rate of an Aurora Serverless
+   v2 instance depends on its current capacity. The higher the current capacity,
+   the faster it can scale up. So for example, if you need to very quickly scale
+   up to high capacity, you might have to set the minimum higher to allow for
+   faster scaling.
+3. If you have a higher amount of data in the buffer cache you may have to set
+   the minimum higher so that data is not evicted from the buffer cache when
+   then instances scale down.
+4. Some features require certain minimums
+    - Performance insights - 2 ACUs
+    - Aurora global databases - 8 ACUs (in primary region)
+5. For high write workloads with serverless v2 readers in tiers 2-15 (more on
+   tiers below), you may need to provision a higher minimum capacity to reduce
+   replica lag.
+     - When using a provisioned writer, you should set the minimum capacity to a
+       value that represents a comparable amount of memory and CPU to the
+       writer. (this is only if you notice replication lag)
+
+
+Another way that you control the capacity/scaling of your serverless v2 reader
+instances is based on the [promotion tier](https://aws.amazon.com/blogs/aws/additional-failover-control-for-amazon-aurora/)
+which can be between 0-15. Previously with all provisioned clusters this didn't
+matter too much (we set everything to 1), but with serverless v2 it starts to
+matter. Any serverless v2 instance in the 0-1 tiers will scale alongside the
+writer even if the current read load does not require the capacity. This is
+because instances in the 0-1 tier are first priority for failover and Aurora
+wants to ensure that in the event of a failover the reader that gets promoted is
+scaled to handle the write load.
+
+- Serverless v2 readers in tier 0-1 scale with the writer
+- Serverless v2 readers in tier 2-15 scale with the read load on itself.
+
+### Serverless v2 vs Provisioned
+
+Serverless v2 can go up to 256GB memory. Anything above that needs to be provisioned.
+   - For example, if the CPU utilization for a db.r6g.4xlarge (128 GB)
+     instance stays at 10% most times, then the minimum ACUs may be set
+     at 6.5 ACUs (10% of 128 GB) and maximum may be set at 64 ACUs (64x2GB=128GB).
+   - But if using a db.r6g.16xlarge (512 GB) a serverles v2 instance can't scale
+     to match
+
+Out of the 4 common usage patters (peaks & valleys, outliers, random, and
+consistent), provisioned should be preferred for the consistent pattern. With
+mixed use cluster you may have writers/readers that follow different patters.
+For example, you may have a writer that follows the `consistent` pattern, but a
+reader that follows the `outliers` pattern. In this case you may want to have a
+provisioned instance as the writer along with a provisioned writer in tier 0-1
```

Comment:
```suggestion
provisioned instance as the writer along with a provisioned reader in tier 0-1
```

Right? Because the reader will become a writer in event of failover?